packages/builder/src/components/design/PropertiesPanel/DesignView.svelte

```diff
@@ -64,10 +69,20 @@
       {/if}
     </div>
   </div>
+  {#if componentDefinition?.transitionable}
+     <div class="transitions">
+       <Select value={componentInstance._transition || ""} on:change={event => onUpdateTransition(event.target.value)} name="transition" label="Transition" secondary thin>
+         <option value="">Choose a transition</option>
```

Comment:
Should we rename the placeholder to `None` to make it more obvious that you can choose to have no transitions?

Reply:
Yep, I think that makes sense.